### ISSUE TITLE
Time for Quartz cron set improperly

### ISSUE THREAD

**Issue by jmrozanec** [jmrozanec/cron-utils#123]
Hi,

the next execution time is set improperly if 5/15 expression is used in Quartz cron. I have a quartz cron expression: "0 5/15 \* \* \* ? *". 
I have set time as: 2016-07-30T15:00:00.000000527Z. 
The Result: 
nextExecution: "2016-07-30T15:15Z"
lastExecution: "2016-07-30T14:45Z"
Expected Result would be: 
nextExecution: "2016-07-30T15:05Z"
lastExecution: "2016-07-30T14:50Z"

Also in current version lastExecution time if cron expression would be "0 0/15 \* \* \* ? *" is returned wrong. Cause it should be "2016-07-30T15:00Z"..

Sample code:

```
CronParser quartzCronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(QUARTZ));
String quartzCronExpression2 = "0 5/15 * * * ? *";
Cron parsedQuartzCronExpression = quartzCronParser.parse(quartzCronExpression2);

ExecutionTime executionTime = ExecutionTime.forCron(parsedQuartzCronExpression);

ZonedDateTime zonedDateTime = LocalDateTime.of(2016, 7, 30, 15, 00, 0, 527).atZone(ZoneOffset.UTC);

ZonedDateTime nextExecution = executionTime.nextExecution(zonedDateTime);
ZonedDateTime lastExecution = executionTime.lastExecution(zonedDateTime);
```

**Comment by hubsn** [comment]
I think same issue with #124. I wrote #124 before read this.

**Comment by jmrozanec** [comment]
@hubsn thanks! If you have some time to write a test or even contribute a solution to it, I will really appreciate. Thanks!

**Comment by jmrozanec** [comment]
@hubsn Fixed! We expect to have a new release soon with this fix.

**Comment by jmrozanec** [comment]
@hubsn The fix was released with version 5.0.5.
